### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by lengushuru**
**In this PR the following changes were made**

- Create To-Do List structure
- Fix Linters errors
- Follow GitHub Flow
- Fixed npm start configuration problem.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  {
+    description: 'call mum',
+    completed: false,
+    index: 3,
+  },
+
+];
+
+tasks.sort((a, b) => a.index - b.index);
+
```
Comment (joskalenda) [inline src/index.js:20]: - [ ] I have checked your code but am not able to run the app in my locally, see the image below and kindly fix this and resubmit your work

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @lengushuru , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
| |

## Highlights

- [ ]... ✅ Good commit messages
- [ ].. ✅ Green linters
- [ ].... ✅ Great code syntax and clear

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -25,6 +41,256 @@ OPTIONAL SECTIONS:
...
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
```
Comment (Qoosim) [inline README.md:293]: - [OPTIONAL] You have duplicate of your readme. Kindly remove the duplicate. The top README is fine.

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @lengushuru, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**